airflow/api/common/experimental/trigger_dag.py

```diff
@@ -82,6 +82,7 @@ def _trigger_dag(
     for _dag in dags_to_trigger:
         trigger = _dag.create_dagrun(
             run_id=run_id,
+            start_date=timezone.utcnow(),
```

Comment:
If the dag you trigger is timezone aware does it matter?
I mean you consider here the default timezone but the dag you are triggering may be in a diffrent timezone than the default

Reply:
The `start_date` here is not the DAG start_date. It is the dag run starting time of the run. Internally it's all tracked in UTC and rendered in the user's timezone on UI.

Correct me if I am wrong.

Reply:
You’re correct. Although maybe we shouldn’t set this to `RUNNING` in the first place, but should set it to `QUEUED` and let the scheduler start the run (and set `start_date`) instead. See how this is implemented in the new REST API (in `airflow/api_connexion/endpoints/dag_run_endpoint.py::post_dag_run`).

Reply:
Supported. Setting it to queued will save us some complaints especially about max_active_runs

Reply:
Should I go ahead and make that change? There are 3 places where we are creating dag_runs with the state running, should this be the same behavior across the board?

![image](https://user-images.githubusercontent.com/10116000/133594527-9f975155-09a9-431f-a462-17f98236177c.png)

Reply:
This change will affect three places. All these use the `trigger_dag` function. 

1. TriggerDAGRunOperator
2. APIClient - `dag_runs`
3. CLI trigger dag

Is that okay?